### PR TITLE
Release 0.11.0

### PR DESCRIPTION
**Pull request by oeble**
Note: temporarily disable the twap validation to allow for points accumulation.

### CODE REVIEW

File: programs/scope/src/oracles/twap.rs [hunk 1]
```diff
@@ -66,8 +66,8 @@ pub fn get_price(
         .get(source_index)
         .ok_or(ScopeError::TwapSourceIndexOutOfRange)?;
 
-    let current_ts = clock.unix_timestamp.try_into().unwrap();
-    utils::validate_ema(twap, current_ts)?;
+    // let current_ts = clock.unix_timestamp.try_into().unwrap();
```
Comment (silviutroscot) [inline programs/scope/src/oracles/twap.rs:69]: why did you remove this?

Comment (oeble) [reply programs/scope/src/oracles/twap.rs:69]: Because otherwise all scope twap would have been old for one hour after the update. We will do a second update to activate the check.

**Review by silviutroscot — APPROVED** [review]
